### DISCUSSION TITLE
Is there a way to change the default browser that opens links?

### DISCUSSION THREAD

**Discussion by ealexandrohin** [vercel/hyper#6793] [Q&A]
^^^ And how do you do it?

**Comment by f3bw** [comment]
Try the following:

1. Open the .hyper.js configuration file in your default text editor.
2. Scroll down or search for the shell property in the configuration file. It should look something like this:
`shell: 'C:\\WINDOWS\\System32\\bash.exe',`

3. Replace the current shell command with the command to open your desired browser. For example, if you want to use Google Chrome, the configuration would look like this:
`shell: 'C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe',`
4. Save the changes and close the configuration file.

Restart Hyper terminal for the changes to take effect.

**Reply by ealexandrohin** [reply]
Now, it just opens Chrome every time and closes Hyper immediately.
